addons/website_membership/controllers/main.py

```diff
@@ -134,34 +123,30 @@ def members(self, membership_id=None, country_name=None, country_id=0, page=1, *
                     free_end = max(offset + limit - count_members, 0)
                     memberships_partner_ids['free'] = free_partner_ids[free_start:free_end]
                     page_partner_ids |= set(memberships_partner_ids['free'])
-                google_map_partner_ids += free_partner_ids[:2000-len(google_map_partner_ids)]
+                google_map_partner_ids += free_partner_ids[:2000 - len(google_map_partner_ids)]
                 count_members += len(free_partner_ids)
 
         google_map_partner_ids = ",".join(map(str, google_map_partner_ids))
 
-        partners = { p.id: p for p in partner_obj.browse(request.cr, SUPERUSER_ID, list(page_partner_ids), request.context)}
+        partners = {p.id: p for p in Partner.sudo().browse(list(page_partner_ids))}
```

Comment:
`list()` call is unnecessary, `browse()` converts iterables to tuples during input normalisation